Adapter/SymfonyCache.php

```diff
@@ -141,7 +146,10 @@ public function flush(array $keys = ['all']): bool
                 if ($result) {
                     $result = substr($content, -2) == 'ok';
                 } else {
-                    return $content;
+                    throw new \UnexpectedValueException(sprintf(
```

Comment:
do we need a linebreak here?

Reply:
`sprintf` may be avoid here. We should discuss about it on dev-kit.